Source/Core/Heap.js

```diff
@@ -111,7 +111,7 @@ define([
         var data = this._data;
         var candidate = -1;
 
-        while (true) {
+        while (true) { // eslint-disable-line no-constant-condition
```

Comment:
I'm not sure I like the idea of `while(true)` in our code to begin with, but I guess this can be discussed in the 3d-tiles PR itself.